src/types/types/resources.schema.d.ts

```diff
@@ -100,27 +106,54 @@ export type OpsgeniePriorityLevel = 'P1' | 'P2' | 'P3' | 'P4' | 'P5';
 export type PagerDutyEventType = 'change' | 'alert';
 export type PagerDutyEventAction = 'trigger' | 'acknowledge' | 'resolve';
 export type PagerDutySeverity = 'critical' | 'error' | 'warning' | 'info';
-export type CategoryNotificationIds = Notification1[];
+export type DefenderID2 = string;
+export type CategoryOrDefenderID = Category | DefenderID3;
+export type NotificationOrDefenderID1 = Notification | DefenderID2;
+export type CategoryNotificationIds = NotificationOrDefenderID1[];
+export type DefenderID3 = string;
+export type RelayerOrDefenderID1 = Relayer | DefenderID;
```

Comment:
should the "1" at the end be there?

Reply:
Good catch. These are auto-generated by json-schema for documentation